src/utils/customMessages.js

```diff
@@ -97,6 +97,11 @@ const messages = {
   viewStatsUnauthorized: 'Permission denied trying to view trips stats of another user, you must be the owner to view the stats',
   viewStatsNoRequesterId: 'Please specify a valid value for the requester id, it must be a positive number',
   requesterNotRegistered: 'Sorry, we could not find a user account associated with that requesterId',
+  requestUpdated: 'The request is updated successfully',
+  notOpenRequest: 'You can not updated rejected or approved travel request',
```

Comment:
Could you correct this to `You cannot update a rejected or approved travel request`?